etc/.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[tool.isort]
```

Comment:
Should we align more closely with https://raw.githubusercontent.com/github/super-linter/main/TEMPLATES/.isort.cfg ?

Reply:
I think the answer is no, that the profile=black is simply a better approach to what they have upstream in super-linter

Reply:
Yeah, isort specifically has a profile for black compatibility that they maintain. It's probably easier to let them handle the compatibility and just leverage their work for the profile.